ACORN/current_generator/wera.py

```diff
@@ -66,6 +66,19 @@ def has_enough_radials(radial_file_list):
         return retval
 
     @staticmethod
+    def has_current_data(nc_file):
+        # Return True if netcdf file includes at least 1 current measurement != NaN
+        retval = True
+        F = Dataset(nc_file, mode='r')
+        if F['UCUR'][:].mask.all() & F['VCUR'][:].mask.all():
```

Comment:
It may not make a difference in some cases, but I think you want the boolean operator `and` here.
`&` means *bitwise* AND.